config/jobs/kubernetes/sig-gcp/sig-gcp-gke-config.yaml

```diff
@@ -255,10 +255,10 @@ periodics:
       - --extract=ci/latest
       - --gcp-cloud-sdk=gs://cloud-sdk-testing/ci/staging
       - --gcp-node-image=gci
-      - --gcp-zone=us-central1-f
+      - --gcp-zone=us-west1-b
       - --ginkgo-parallel
       - --gke-environment=test
-      - --gke-node-locations=us-central1-f,us-central1-a,us-central1-b
+      - --gke-node-locations=us-west1-b,us-central1-a,us-central1-b
```

Comment:
these probably need to be changed as a group as well

Reply:
hmm, I'm not sure on the details of these multizone jobs, but perhaps I'll revert changing these tests? or should we hope they would work similarly in a different region?

Reply:
neither am I, no preference which you do, but I would leave them all in a single region